pkg/webui/console/components/rights-group/index.js

```diff
@@ -118,26 +146,43 @@ class RightsGroup extends React.Component {
           key={right}
           name={right}
           label={{ id: `enum:${right}` }}
+          disabled={outOfOwnScopeRights.includes(right)}
```

Comment:
What is the purpose of this? I can agree that is looks nice, but the user with less rights cannot modify another collaborator that has more rights. The IS would return a 403 error. 
<img width="644" alt="Screenshot 2019-06-26 at 13 42 44" src="https://user-images.githubusercontent.com/16374166/60177104-52f5fa80-9818-11e9-869d-ba6502679230.png">

In this case the current user has only view application/edit application/view and edit app collaborators rights, while `test-user2` has more.

Reply:
Yeah, this is actually more complex than thought. The API behavior is in my eyes not optimal (see #857). Given the *edge-casyness* of this thing. I would go for just disabling altering any rights if the given collaborator has more rights than oneself.

Reply:
I have added a `strict` prop to the `<RightsGroup />` that will prevent any rights modification of users with more rights. The whole scenario is too much of an edge case to go on with it at the moment. We should pick this up later on but we need a proper strategy for communicating this constraint to the user. The logic we have now is a good start to do this,.